drivers/include/ds3234.h

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2018 SKF AB
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    drivers_ds3234  DS3234 Extremely Accurate SPI RTC
```

Comment:
@gebart: I think the `@ingroup` is missing here. As a documentation is shown [directly in modues](http://api.riot-os.org/modules.html) in the doxygen doc. Care to make a follow up PR to address this?